### PR TITLE
german locale

### PR DESCRIPTION
**Pull request by benjamintd**
Right now it's just the raw output of Deepl. @mwkroening since you submitted a german language PR recently, perhaps you are interested in reviewing this one ? It's probably highly inconsistent and wrong, but my german classes are very far away 😅.

### CODE REVIEW

**Comment by mwkroening** [comment]
Awesome! I'll try to review this later today.

**Comment by benjamintd** [comment]
Awesome. 
Here is the preview link to see the translations in context: https://codenames-fhlr66zip.vercel.app/de 
Let me know how I can help you :)

File: locales/de.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  blue: "Blau",
+  duetA: "Seite A",
+  duetB: "Seite B",
+  spectator: "Zuschauer",
+  "just-joined": "hat dem Spiel beigetretet!",
+  "game-created": "Das Spiel wurde erstellt",
+  "no-players-yet": "Es gibt noch keine Spieler in diesem Team!",
+  "player-name-placeholder": "Spielername eingeben",
+  cards: "Karten",
+  buymeacoffee: "Unterstützen Sie das Spiel, kaufen Sie mir einen Kaffee",
```
Comment (mwkroening) [inline locales/de.ts:68]:
```suggestion
  buymeacoffee: "Unterstütze das Spiel, kaufe mir einen Kaffee",
```

File: locales/de.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  "positive-reaction-1": "Schön!",
+  "positive-reaction-2": "Gut gemacht!",
+  "positive-reaction-3": "Starker Zug!",
+  "positive-reaction-4": `Ein Punkt!`,
+  "positive-reaction-5": "Juhu!",
+  "positive-reaction-6": "😎",
+  "neutral-reaction-1": "Das ist ein Fehlschlag.",
```
Comment (mwkroening) [inline locales/de.ts:75]:
```suggestion
  "neutral-reaction-1": "Das ging daneben.",
```

File: locales/de.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  "neutral-reaction-2": "Nahe genug.",
```
Comment (mwkroening) [inline locales/de.ts:76]: Not sure, what this is supposed to mean.

File: locales/de.ts [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  "neutral-reaction-3": "Nächstes Mal schaffst du es.",
+  "neutral-reaction-4": "Das war ein Zivilist.",
```
Comment (mwkroening) [inline locales/de.ts:78]:
```suggestion
  "neutral-reaction-4": "Das war ein unbeteiligter Zuschauer.",
```

File: locales/de.ts [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  "neutral-reaction-5": "Fast!`",
+  "neutral-reaction-6": "Zu schade.",
+  "black-reaction-1": "Sie haben gerade das Spiel verloren!",
```
Comment (mwkroening) [inline locales/de.ts:81]:
```suggestion
  "black-reaction-1": "Ihr habt gerade das Spiel verloren!",
```

File: locales/de.ts [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  "black-reaction-2": "Die andere Mannschaft gewinnt!",
```
Comment (mwkroening) [inline locales/de.ts:82]:
```suggestion
  "black-reaction-2": "Das andere Team gewinnt!",
```

File: locales/de.ts [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  "black-reaction-3": "Ihre Mannschaft verliert ☠️",
```
Comment (mwkroening) [inline locales/de.ts:83]:
```suggestion
  "black-reaction-3": "Dein Team verliert ☠️",
```

File: locales/de.ts [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  "black-reaction-4": "Oh nein, Sie haben auf einen Attentäter geklickt ☠️",
```
Comment (mwkroening) [inline locales/de.ts:84]:
```suggestion
  "black-reaction-4": "Oh nein, du hast auf einen Attentäter geklickt ☠️",
```

File: locales/de.ts [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+  "negative-reaction-1": "Hoppla!",
```
Comment (mwkroening) [inline locales/de.ts:85]:
```suggestion
  "negative-reaction-1": "Ups!",
```

File: locales/de.ts [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+  "negative-reaction-2": "Oh nein!",
+  "negative-reaction-3": `Das andere Team dankt Ihnen.`,
```
Comment (mwkroening) [inline locales/de.ts:87]:
```suggestion
  "negative-reaction-3": `Das andere Team dankt euch.`,
```

File: locales/de.ts [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+  "negative-reaction-4": "Keine Sorge, Sie haben Ihr Bestes versucht 💛.",
```
Comment (mwkroening) [inline locales/de.ts:88]:
```suggestion
  "negative-reaction-4": "Keine Sorge, Ihr haben euer Bestes gegeben. 💛",
```

File: locales/de.ts [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+  "negative-reaction-5": "Keine Sorge, Sie haben Ihr Bestes gegeben. 🤦",
```
Comment (mwkroening) [inline locales/de.ts:89]:
```suggestion
  "negative-reaction-5": "🤦",
```

File: locales/de.ts [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-classic-teams": `Die Spieler werden in zwei Mannschaften aufgeteilt: <1>rot</1> und <3>blau</3>. Ein Spieler jeder Mannschaft wird zum Spymaster der Mannschaft gewählt 🕵️ - die anderen sind Feldspieler`,
```
Comment (mwkroening) [inline locales/de.ts:90]:
```suggestion
  "rules-classic-teams": `Die Spieler werden in zwei Teams aufgeteilt: <1>rot</1> und <3>blau</3>. Ein Spieler jedes Teams wird zu dessen Geheimdienstchef gewählt 🕵️ - alle anderen sind Ermittler.`,
```

File: locales/de.ts [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-classic-grid": `Fünfundzwanzig Wortkarten werden zufällig in einem Gitter ausgelegt. Eine Anzahl dieser Wörter repräsentieren <1>rot</1> Agenten, eine Anzahl repräsentiert <3>blau</3> Agenten, eines repräsentiert einen <5>Meuchelmörder</5>, und die anderen repräsentieren <7>unschuldige Zuschauer</span>. Die Geheimdienstchef der Teams kennen die Farben der einzelnen Wörter und weisen ihr Team abwechselnd darauf hin, damit sie die Wörter ihrer Farbe finden.`,
```
Comment (mwkroening) [inline locales/de.ts:91]:
```suggestion
  "rules-classic-grid": `25 zufällige Wortkarten werden in einem 5-mal-5-Raster ausgelegt. Ein Teil dieser Wörter steht für <1>rote</1> Agenten, ein anderer Teil für <3>blaue</3> Agenten, eines für einen <5>Attentäter</5>, und die anderen repräsentieren <7>unbeteiligte Zuschauer</span>. Die Geheimdienstchefs der Teams kennen die Farben der einzelnen Wörter und geben ihrem Team abwechselnd Hinweise, damit sie die Wörter ihrer Farbe erraten.`,
```

File: locales/de.ts [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-classic-hints": `Jeder Hinweis darf nur aus einem einzigen Wort und einer Anzahl von Karten bestehen, auf die er sich bezieht. Der Spielleiter gibt einen Hinweis, der sich auf möglichst viele der Wörter auf den Karten seiner eigenen Agenten bezieht, aber nicht auf andere - er könnte sein Team versehentlich dazu bringen, eine Karte zu wählen, die einen unschuldigen Zuschauer, einen gegnerischen Agenten oder den Attentäter darstellt.`,
```
Comment (mwkroening) [inline locales/de.ts:92]:
```suggestion
  "rules-classic-hints": `Jeder Hinweis darf nur aus einem einzigen Wort und einer Anzahl von Karten bestehen, auf die er sich bezieht. Der Geheimdienstchef gibt einen Hinweis, der sich auf möglichst viele der Wörter auf den Karten seiner eigenen Agenten bezieht, aber nicht auf andere - er könnte sein Team versehentlich dazu bringen, eine Karte zu wählen, die einen unbeteiligten Zuschauer, einen gegnerischen Agenten oder den Attentäter darstellt.`,
```

File: locales/de.ts [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-hint-restrictions": `Das Wort des Hinweises kann frei gewählt werden, solange es keines der Wörter auf den Codenameskarten ist (und nicht enthält), die zu diesem Zeitpunkt noch auftauchen. Phonetische Hinweise sind typischerweise verboten.`,
```
Comment (mwkroening) [inline locales/de.ts:93]:
```suggestion
  "rules-hint-restrictions": `Das Hinweiswort kann frei gewählt werden, solange es keines der Wörter ist (und nicht enthält), die zu diesem Zeitpunkt noch offen sind. Phonetische Hinweise sind typischerweise verboten.`,
```

File: locales/de.ts [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-turns": `Die Karten werden aufgedeckt, wenn Vermutungen angestellt werden. Außendienstmitarbeiter müssen mindestens eine Vermutung pro Runde anstellen, wobei sie eine falsche Vermutung und deren Folgen riskieren. Sie können ihren Zug auch zu jedem späteren Zeitpunkt freiwillig beenden. Sie können bis zu der vom Geheimdienstchef angegebenen Zahl plus eins (ein Bonus, um ein zuvor angedeutetes Wort zu finden) raten`,
```
Comment (mwkroening) [inline locales/de.ts:94]:
```suggestion
  "rules-turns": `Die Karten werden aufgedeckt, wenn Rateversuche werden. Ermittler müssen mindestens einen Rateversuch pro Runde tätigen, wobei sie einen Fehlschlag riskieren. Sie können ihren Zug auch zu jedem späteren Zeitpunkt freiwillig beenden. Sie können bis zu der vom Geheimdienstchef angegebenen Zahl plus eins (ein Bonus, um ein zuvor angedeutetes Wort zu finden) raten.`,
```

File: locales/de.ts [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-classic-game-end": `Das Spiel endet, wenn alle Agenten einer Mannschaft identifiziert sind (🏅), oder wenn eine Mannschaft den <1>Assassinen</1> identifiziert hat (☠️).`,
```
Comment (mwkroening) [inline locales/de.ts:95]:
```suggestion
  "rules-classic-game-end": `Das Spiel endet, wenn alle Agenten einer Mannschaft identifiziert sind (🏅), oder wenn eine Mannschaft den <1>Attentäter</1> identifiziert hat (☠️).`,
```

File: locales/de.ts [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-duet-objective": `Zwei Spieler arbeiten zusammen, um 15 Wörter auf der Tafel zu finden.`,
```
Comment (mwkroening) [inline locales/de.ts:96]:
```suggestion
  "rules-duet-objective": `Zwei Spieler arbeiten zusammen, um 15 Wörter auf dem Raster zu finden.`,
```

File: locales/de.ts [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-duet-grid": `Fünfundzwanzig Wortkarten werden zufällig in einem Raster angeordnet. Eine Anzahl dieser Wörter steht für <1>grün</1> Agenten, eine Anzahl steht für <3>Meuchelmörder</3>, und die anderen stehen für <5>unschuldige Zuschauer</5>. Jeder Spieler kennt 9 <7>grüne</7> Wörter, von denen sich einige mit der anderen Seite überschneiden`,
```
Comment (mwkroening) [inline locales/de.ts:97]:
```suggestion
  "rules-duet-grid": `25 zufällige Wortkarten werden in einem 5-mal-5-Raster ausgelegt. Ein Teil dieser Wörter steht für <1>grüne</1> Agenten, ein anderer für <3>Attentäter</3>, und die restlichen für <5>unschuldige Zuschauer</5>. Jeder Spieler kennt 9 <7>grüne</7> Wörter, von denen sich einige mit der anderen Seite überschneiden.`,
```

File: locales/de.ts [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-duet-hints": `Die Spieler geben abwechselnd Hinweise, damit der andere Spieler seine grüne Karte findet. Jeder Hinweis darf nur aus einem einzigen Wort und einer Anzahl von Karten bestehen, auf die er sich bezieht. Der Geheimdienstchef gibt einen Hinweis, der sich auf möglichst viele der Wörter auf den Karten seiner eigenen Agenten bezieht, aber nicht auf andere - sie könnten den anderen Spieler versehentlich dazu verleiten, eine Karte zu wählen, die einen unschuldigen Zuschauer oder einen Attentäter darstellt`,
```
Comment (mwkroening) [inline locales/de.ts:98]:
```suggestion
  "rules-duet-hints": `Die Spieler geben abwechselnd Hinweise, damit der andere Spieler seine grüne Karte errät. Jeder Hinweis darf nur aus einem einzigen Wort und einer Anzahl von Karten bestehen, auf die er sich bezieht. Der Geheimdienstchef gibt einen Hinweis, der sich auf möglichst viele der Wörter auf den Karten seiner eigenen Agenten bezieht, aber nicht auf andere - sie könnten den anderen Spieler versehentlich dazu verleiten, eine Karte zu wählen, die einen unschuldigen Zuschauer oder einen Attentäter darstellt`,
```

File: locales/de.ts [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+  "rules-duet-game-end": `Das Spiel endet, wenn alle 15 grünen Agenten identifiziert sind (🏅), oder wenn ein Spieler einen <1>Assassinen</1> identifiziert hat (☠️). Das Spiel ist zeitgesteuert, was bedeutet, dass die Spieler insgesamt 9 Runden Zeit haben, um alle Wörter zu finden. Ein Klick auf einen <3>Unschuldiger Zuschauer</Spanne> verbraucht einen zusätzlichen Zug`,
```
Comment (mwkroening) [inline locales/de.ts:99]:
```suggestion
  "rules-duet-game-end": `Das Spiel endet, wenn alle 15 grünen Agenten identifiziert sind (🏅), oder wenn ein Spieler einen <1>Assassinen</1> identifiziert hat (☠️). Das Spiel ist zeitbegrenzt. Beide Spieler haben insgesamt 9 Runden Zeit, um alle Wörter zu finden. Ein Klick auf einen <3>unschuldiger Zuschauer</Spanne> verbraucht einen zusätzlichen Zug.`,
```

File: locales/de.ts [hunk 23]
```diff
@@ -0,0 +1,109 @@
...
+  "long-story-short": "Lange Geschichte-kurz",
```
Comment (mwkroening) [inline locales/de.ts:100]:
```suggestion
  "long-story-short": "Langer Rede kurzer Sinn",
```

File: locales/de.ts [hunk 24]
```diff
@@ -0,0 +1,109 @@
...
+  "privacy-tldr": `
+  <p>Die Partien und Chats verbleiben für eine gewisse Zeit in unserer Datenbank, können aber jederzeit gelöscht werden. Dies ist die einzige Information, die wir speichern.</p>
+  <p>Wir verwenden Cookies, um Analysen über die Nutzung der Website (demographische Daten, Dauer der Sitzungen, Art der gespielten Spiele) zu erhalten. Diese Analysen enthalten keine persönlich identifizierbaren Informationen.</p>
+  <p> Unser Analytik - Anbieter ist Google Analytics.< /p>
```
Comment (mwkroening) [inline locales/de.ts:104]:
```suggestion
  <p> Unser Trackingtool ist Google Analytics.< /p>
```

**Review by mwkroening — CHANGES_REQUESTED** [review]
This is a preliminary review.
I suggested many changes and changed the style from “sie” zu “du”.

Generally this is fine, but I could still improve on some wording in the next days. Also, I'll ask a friend to have a look at this as well.

File: locales/de.ts (hunk 23 above)
Comment (benjamintd) [reply locales/de.ts]: this is lovely

**Review by benjamintd — COMMENTED** [review]
This is great, @mwkroening. Thanks so much.

File: locales/de.ts [hunk 25]
```diff
@@ -0,0 +1,108 @@
...
+  "winner-team": "Das <1>{{Team}}</1> Team gewinnt!",
+  "black-card-loss": "Das Spiel ist vorbei. Ihr habt verloren, weil ihr auf eine schwarze Karte geklickt haben.",
+  "out-of-time-loss": "Die Zeit ist abgelaufen.",
+  won: "Dein Team hat gewonnen!",
+  "join-red": "Rot beitreten",
+  "join-blue": "Blau beitreten",
+  "join-side": "Seite beitreten",
+  spectate: "Zuschauen",
+  "game-started": "Das Spiel hat begonnen",
+  "page-title": "Codenames - Online spielen",
```
Comment (mwkroening) [inline locales/de.ts:53]:
```suggestion
  "page-title": "Codenames online spielen",
```

File: locales/de.ts [hunk 26]
```diff
@@ -0,0 +1,108 @@
...
+  "Spiele Codenames online, ein Mehrspieler-Wörterspiel, das für Handys und Desktop verfügbar ist. Bei diesem Ratespiel treten zwei Teams gegeneinander an und versuchen mithilfe von Hinweisen Codewörter zu erraten.",
+  "in-the-room": "Im Raum",
+  rules: "Regeln",
+  red: "Rot",
+  blue: "Blau",
+  duetA: "Seite A",
+  duetB: "Seite B",
+  spectator: "Zuschauer",
+  "just-joined": "ist dem Spiel beigetreten!",
+  "game-created": "Das Spiel wurde erstellt.",
```
Comment (mwkroening) [inline locales/de.ts:64]:
```suggestion
  "game-created": "Spiel erstellt.",
```

File: locales/de.ts [hunk 27]
```diff
@@ -0,0 +1,108 @@
...
+  "rules-classic-hints": `Jeder Hinweis darf nur aus einem einzigen Wort und einer Anzahl von Karten bestehen, auf die er sich bezieht. Der Geheimdienstchef gibt einen Hinweis, der sich auf möglichst viele der Wörter auf den Karten seiner eigenen Agenten bezieht, aber nicht auf andere - er könnte sein Team versehentlich dazu bringen, eine Karte zu wählen, die einen unbeteiligten Zuschauer, einen gegnerischen Agenten oder den Attentäter darstellt.`,
+  "rules-hint-restrictions": `Das Hinweiswort kann frei gewählt werden, solange es keines der Wörter ist (und nicht enthält), die zu diesem Zeitpunkt noch offen sind. Phonetische Hinweise sind typischerweise verboten.`,
+  "rules-turns": `Die Karten werden aufgedeckt, wenn Rateversuche werden. Ermittler müssen mindestens einen Rateversuch pro Runde tätigen, wobei sie einen Fehlschlag riskieren. Sie können ihren Zug auch zu jedem späteren Zeitpunkt freiwillig beenden. Sie können bis zu der vom Geheimdienstchef angegebenen Zahl plus eins (ein Bonus, um ein zuvor angedeutetes Wort zu finden) raten.`,
```
Comment (mwkroening) [inline locales/de.ts:94]:
```suggestion
  "rules-turns": `Die Karten werden aufgedeckt, wenn geraten wird. Ermittler müssen mindestens einen Rateversuch pro Runde tätigen, wobei sie einen Fehlschlag riskieren. Sie können ihren Zug auch zu jedem späteren Zeitpunkt freiwillig beenden. Sie können bis zu der vom Geheimdienstchef angegebenen Zahl plus eins (ein Bonus, um ein zuvor angedeutetes Wort zu finden) raten.`,
```

File: locales/de.ts [hunk 28]
```diff
@@ -0,0 +1,108 @@
...
+  "rules-duet-hints": `Die Spieler geben abwechselnd Hinweise, damit der andere Spieler seine grüne Karte errät. Jeder Hinweis darf nur aus einem einzigen Wort und einer Anzahl von Karten bestehen, auf die er sich bezieht. Der Geheimdienstchef gibt einen Hinweis, der sich auf möglichst viele der Wörter auf den Karten seiner eigenen Agenten bezieht, aber nicht auf andere - sie könnten den anderen Spieler versehentlich dazu verleiten, eine Karte zu wählen, die einen unschuldigen Zuschauer oder einen Attentäter darstellt`,
+  "rules-duet-game-end": `Das Spiel endet, wenn alle 15 grünen Agenten identifiziert sind (🏅), oder wenn ein Spieler einen <1>Assassinen</1> identifiziert hat (☠️). Das Spiel ist zeitbegrenzt. Beide Spieler haben insgesamt 9 Runden Zeit, um alle Wörter zu finden. Ein Klick auf einen <3>unschuldiger Zuschauer</Spanne> verbraucht einen zusätzlichen Zug.`,
```
Comment (mwkroening) [inline locales/de.ts:99]:
```suggestion
  "rules-duet-game-end": `Das Spiel endet, wenn alle 15 grünen Agenten identifiziert sind (🏅), oder wenn ein Spieler einen <1>Attentäter</1> identifiziert hat (☠️). Das Spiel ist zeitbegrenzt. Beide Spieler haben insgesamt 9 Runden Zeit, um alle Wörter zu finden. Ein Klick auf einen <3>unschuldiger Zuschauer</Spanne> verbraucht einen zusätzlichen Zug.`,
```

**Review by mwkroening — CHANGES_REQUESTED** [review]
These are my final remarks.

Otherwise, it looks good to me. Thanks!

**Comment by benjamintd** [comment]
Thank you @mwkroening. I could not have merged it without your help!

**Pull request merged**